cmd/run.go

```diff
@@ -60,11 +60,11 @@ var runCommand = &cobra.Command{
 				volume = user.HomeDir + volume[1:]
 			}
 			dockerArgs = append(dockerArgs, "-v")
-			dockerArgs = append(dockerArgs, volume)
+			dockerArgs = append(dockerArgs, os.ExpandEnv(volume))
 		}
```

Comment:
@nithin-bose I think your use case is still fully supported
Let's consider a dockerfile with
`LABEL io.whalebrew.config.environment '["TERM", "FOOBAR_NAME","FOO=bar", "key=${foo}","${foobar}","${bar}=value]'`
As fas as [os documentation](https://golang.org/pkg/os/#ExpandEnv) says, and according to current implementation, with a run time environment of:
```
{
    "foo": "_foo",
    "foobar": "_foobar",
    "bar": "_bar"
}
```
the resulting docker command will contain:
`-e TERM -e FOOBAR_NAME -e key=_foo -e _foobar -e _bar=value`

By the way, I am curious of your use case. As far as I understand, in your Dockerfile, you are using
`LABEL io.whalebrew.config.environment '["KEY=value']'`
Having a static value stored in the label sounds close to having an environment variable to me:
`ENV KEY=value`
Did I miss anything?